common/app/experiments/Experiments.scala

```diff
@@ -65,3 +65,11 @@ object Garnett extends Experiment(
 ) {
   override def isParticipating[A](implicit request: RequestHeader, canCheck: CanCheckExperiment): Boolean = super.isParticipating || GarnettLaunch.isSwitchedOn
 }
+
+object MoonLambda extends Experiment(
+  name = "moon-lambda",
+  description = "Users in this experiment will see 404 page rendered by a lambda",
+  owners = Seq(Owner.withGithub("siadcock")),
+  sellByDate = new LocalDate(2018, 2, 28),
+  participationGroup = Perc1B
+)
```

Comment:
`//TODO:` Is this the correct way to do server side tests? Is there anything else we need to do?

Reply:
You need to also add the experiment here: https://github.com/guardian/frontend/blob/52bcf89329a7128c7a65b02718f2274aa5b163e8/common/app/experiments/Experiments.scala#L10.

After merging this PR 1% of users will see this page